src/main/java/picard/sam/SamErrorMetric/ReadBaseStratification.java

```diff
@@ -100,6 +107,19 @@ public T stratify(RecordAndOffset recordAndOffset, SAMLocusAndReference locusInf
         abstract T stratify(final SAMRecord sam);
     }
 
+    /**
+     * A simple position based stratifier for cases when only the record suffices
+     */
+    abstract static class PositionBasedStratifier<T extends Comparable<T>> implements RecordAndOffsetStratifier<T> {
```

Comment:
You should be able to clamp down on the template T here and make it an Integer, right? if in the future somebody thinks of non-integer positions, we can open it up again....

Reply:
```suggestion
    abstract static class PositionBasedStratifier implements RecordAndOffsetStratifier<Integer> {
```

Reply:
and this will simplify the downstream

Reply:
why not do this?

Reply:
don't ignore me :-)

Reply:
oops I thought I already changed this! Sorry about that